programmers/p와y의-개수.js

```diff
@@ -0,0 +1,16 @@
+function solution(s){
+    var p = s.match(/p/gi);
+    var y = s.match(/y/gi);
```

Comment:
뭐뭐해서 const로 하는 것이 좋습니다.
```suggestion
    const p = s.match(/p/gi);
    const y = s.match(/y/gi);
```

Reply:
한번 정의되서 값이 변하지 않는 변수는 const
값이 나중에 변한다면 let 이라는 키워드를 사용하는 것이 좋아요😁